testsuite/features/step_definitions/common_steps.rb

```diff
@@ -578,8 +578,14 @@
     puts $proxy.run("zypper mr --enable openSUSE-Leap-#{os_version}-Update")
   else
     arch, _code = $proxy.run('uname -m')
-    puts $proxy.run("zypper mr --enable SLE-#{os_version}-#{arch.strip}-Pool")
-    puts $proxy.run("zypper mr --enable SLE-#{os_version}-#{arch.strip}-Update")
+    # if it is a SLE15, repos are more than one Pool and one Update, they are both for each Module
+    if os_version.include? '15'
+      repos, _code = $proxy.run("zypper lr | grep #{os_version} | cut -d'|' -f2")
```

Comment:
Sorry, if I could be out of context, but should we have only os_version as grep, or we should have SLE-Module- also filtered?

Reply:
It could narrow down the target of the grep to make sure we are managing only `SLE-MODULE` repos. Anyway I don't think there will be any other repos with `15`|`15SP1`|etc listed to be grepped out by that.

Would `repos, _code = $proxy.run("zypper lr | grep 'SLE-Module.*#{os_version}.*' | cut -d'|' -f2")` work then?

Reply:
Yep. So, If you think there will not be other repos, then is fine as it is.
I wasn't sure about what the list contains :)

Reply:
@Bischoff yeah it makes sense. I switched to a more general approach assuming target repos will always have the pattern

`SLE.*#{os_version}-#{arch.strip}.*`

that is the pattern we are already assuming to true a part from the `Pool|Update` suffix in non-SLE15 cases. Btw I believe a more general approach is better here as you suggested.

Reply:
Thank you very much @ncounter!